package.json

```diff
@@ -35,6 +35,7 @@
     "babel-plugin-transform-es2015-classes": "^6.7.7",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
+    "babel-polyfill": "^6.9.0",
```

Comment:
This would mean that babel-polyfill only gets loaded when a user clones this repo and runs `npm install`. It would not be available for someone just installing it via npm. If the polyfill is really required for the execution of this package it must remain as a `peerDependency`.